src/main/java/org/commcare/formplayer/session/FormSession.java

```diff
@@ -236,7 +237,7 @@ private void setupFunctionContext() {
     private void loadInstanceXml(FormDef formDef, String instanceContent) throws IOException {
         StringReader stringReader = new StringReader(instanceContent);
         XFormParser xFormParser = new XFormParser(stringReader);
-        xFormParser.loadXmlInstance(formDef, stringReader);
+        XFormParser.loadXmlInstance(formDef, stringReader);
```

Comment:
I think it's best to revert this formatting change for the context of this PR. It's a good change but it also makes `xFormParser` in the line above unnecessary. And for this PR we will rather have a smaller change footprint even if it means we are ignoring some good code modifications in non relevant existing code.